Makefile

```diff
@@ -13,7 +13,7 @@ PUBLISH?=publish
 BUILD_VERSION ?= $(shell git describe --always --match "v*" | sed 's/v//')
 
 # TKG Version
-TKG_VERSION ?= v1.8.0+vmware.1
+TKG_VERSION ?= v1.9.0+vmware.1
```

Comment:
do we use tkg 2.x now?

Reply:
We don't, because we don't have that much big changes to update the major release version.